templates/registration/email_change.html

```diff
@@ -0,0 +1,23 @@
+{% extends "base.html" %}
```

Comment:
Would it be possible to see a screenshot of these emails (mostly to check they render correctly and don't have funny messed up links.)

Reply:
I've attached some screenshots of both the HTML email and the plaintext email.